.travis.yml

```diff
@@ -19,7 +50,9 @@ script:
   - make cyclomatic-check
   - make test
   - make images
+  - sudo make images BUILDER=buildah
```

Comment:
what kind of errors it produce if it runs without sudo ?

Reply:
It produces this error on the distros with with old shadow-utils like OpenSUSE:
ERRO[0000] error reading allowed ID mappings: error reading subuid mappings for user "ed" and subgid mappings for group "ed": open /etc/subuid: no such file or directory

and this one on Ubuntu 16.04 LTS:
[31mERRO[0m[0000] flag provided but not defined: -rootless     
flag provided but not defined: -rootless

I'm planning to wait for the next buildah release and try again. It could work with buildah and run c built from master branch, but I didn't want to use that approach. Latest release of buildah runs runc with unrecognised option as far as I can see.